src/egon/data/processing/heat_demand_timeseries/HTS.py

```diff
@@ -0,0 +1,1203 @@
+
+        heat_demand_profile_zensus.reset_index(inplace=True)
+        
+        heat_demand_profile_zensus = pd.merge(heat_demand_profile_zensus,annual_demand[['zensus_population_id','demand']],
+                             how='inner', on = 'zensus_population_id') 
+
+        heat_demand_profile_zensus.set_index('zensus_population_id',inplace =True)
+        heat_demand_profile_zensus =  heat_demand_profile_zensus[heat_demand_profile_zensus.columns[:-1]].multiply(heat_demand_profile_zensus.demand,axis=0)
+
+    return heat_demand_profile_dist, heat_demand_profile_mv, heat_demand_profile_zensus
+
+
+
+def cts_demand_per_aggregation_level(aggregation_level):
+    '''
+    
+    Description: Create dataframe assigining the CTS demand curve to individual zensus cell
+    based on their respective NUTS3 CTS curve
+
+    Parameters
+    ----------
+    aggregation_level : str
+        if further processing is to be done in zensus cell level 'other'
+        else 'dsitrict'
+
+    Returns
+    -------
+    CTS_per_district : pandas.DataFrame
+        if aggregation ='district'
+            NUTS3 CTS profiles assigned to individual 
+            zensu cells and aggregated per district heat area id
+        else
+            empty dataframe
+    CTS_per_grid : pandas.DataFrame
+        if aggregation ='district'
+            NUTS3 CTS profiles assigned to individual 
+            zensu cells and aggregated per mv grid subst id
+        else
+            empty dataframe
+    CTS_per_zensus : pandas.DataFrame
+        if aggregation ='district'
+            empty dataframe
+        else
+            NUTS3 CTS profiles assigned to individual 
+            zensu population id
+
+    '''
+
+    nuts_zensus = psycop_gdf_AF('boundaries.egon_map_zensus_vg250', geom_column = 'zensus_geom')
+    nuts_zensus.drop('zensus_geom', axis=1, inplace=True)
+
+    demand = psycop_df_AF('demand.egon_peta_heat')
+    demand = demand[(demand['sector']=='service') & (demand['scenario']=='eGon2035')]
+    demand.drop(demand.columns.difference(['demand', 'zensus_population_id']),axis=1,inplace=True)
+    demand.sort_values('zensus_population_id',inplace=True)
+
+    demand_nuts = pd.merge(demand, nuts_zensus, how='left', on = 'zensus_population_id')
+    
+    mv_grid = psycop_df_AF('boundaries.egon_map_zensus_grid_districts')
+    
+    if os.path.isfile('CTS_heat_demand_profile_nuts3.csv'):
+        df_CTS_gas_2011 = pd.read_csv('CTS_heat_demand_profile_nuts3.csv',index_col=0)
+        df_CTS_gas_2011.columns.name = 'ags_lk'
+        df_CTS_gas_2011.index = pd.to_datetime(df_CTS_gas_2011.index)
+        df_CTS_gas_2011 =  df_CTS_gas_2011.asfreq('H')
+    else:
+        df_CTS_gas_2011 = temporal.disagg_temporal_gas_CTS(use_nuts3code=True, year=2011)
+        df_CTS_gas_2011.to_csv('CTS_heat_demand_profile_nuts3.csv')
+           
+    ags_lk = pd.read_csv(os.path.join(os.getcwd(),'t_nuts3_lk.csv'),index_col =0)
```

Comment:
This file is part of the disaggregator tool and can be accessed from there:
`pd.read_csv(os.path.join(os.getcwd(),'/demandregio-disaggregator/disaggregator/disaggregator/data_in/regional/t_nuts3_lk.csv'),index_col =0)`
I would prefer to use this file instead of adding the same csv file again to egon-data.